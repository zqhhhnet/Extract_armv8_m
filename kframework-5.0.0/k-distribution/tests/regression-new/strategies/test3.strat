^ regular ; ^ a2c ; ^ c2a
