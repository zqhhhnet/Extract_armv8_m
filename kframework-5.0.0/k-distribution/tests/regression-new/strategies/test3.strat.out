<wrapper>
  <k>
    a
  </k>
  <trace>
    ListItem ( ~ regular )
    ListItem ( ~ a2c )
    ListItem ( ~ c2a )
  </trace>
  <s>
    #STUCK ( )
  </s>
</wrapper>
